src/utils/date.ts

```diff
@@ -6,7 +6,7 @@ export const parseDate = (date: string) =>
 export const formatDate = (date: string | DateTime) => {
   // ISO String
   if (typeof date === 'string') {
-    return DateTime.fromISO(date).toFormat('MMMM d yyyy');
+    return parseDate(date).toFormat('MMMM d yyyy');
```

Comment:
Can we do this somewhere else? We need utility functions that format UTC dates and another that takes timezone into account.

Since `formatDate` is used throughout the application, it'd be less confusing if we added a new function named `formatUTCDate` (or similar). This might also be a good time to rename `parseDate`.